configuration/drugs/drugs.csv

```diff
@@ -33,7 +34,7 @@ Uuid,Void/Retire,Name,Concept Drug,Concept Dosage Form,Strength,_version:1
 ,,Doxycycline 100mg Tablet,Cambodia_Doxycycline,Tablet,100mg,
 ,,Erythromycin 500mg Tablet,Cambodia_Erythromycin,Tablet,500mg,
 ,,Erythromycin 250mg Tablet,Cambodia_Erythromycin,Tablet,250mg,
-,,Folic acid 220mg Tablet,Cambodia_Folic acid,Tablet,220mg,
+,,Folic acid 5mg Tablet,Cambodia_Folic acid,Tablet,5mg,

```

Comment:
@ouiliam , renaming this drug will create a duplicate when deployed on the production server.
Could you enforce the editing of this drug line by inputting the UUID the existing drug has in production?